OSX with no lzma.h gives os/lzma_stub.h file not found on pip install only

For some reason when I try and build from current pip pysam release on OS X without lzma.h specifically installed I get the following error, I think pysam might be forgetting to copy the lzma_stub into the wheel build dir?

```
  gcc -fPIC -fno-strict-aliasing -fno-common -dynamic -arch i386 -arch x86_64 -O2 -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam/htslib -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam/samtools -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam/samtools/lz4 -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam/bcftools -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam/pysam -I/private/var/folders/03/3h2pdngd2k3d9_h3cb_mv554000dhq/T/pip-build-VjPOxm/pysam -I/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c htslib/cram/cram_io.c -o build/temp.macosx-10.5-intel-2.7/htslib/cram/cram_io.o -Wno-unused -Wno-strict-prototypes -Wno-sign-compare -Wno-error=declaration-after-statement
  htslib/cram/cram_io.c:63:10: fatal error: 'os/lzma_stub.h' file not found
  #include "os/lzma_stub.h"
           ^
  1 error generated.
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for pysam
```
Curiously I don't get this error when I build using setup.py.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -50,8 +50,8 @@ include htslib/htslib.pc.in
 include htslib/htslib/*.h
 include htslib/cram/*.c
 include htslib/cram/*.h
-include htslib/win/*.c
-include htslib/win/*.h
+include htslib/os/*.c
+include htslib/os/*.h
 include cy_build.py
 include pysam.py
 include requirements.txt

